.travis.yml

```diff
@@ -28,5 +28,5 @@ script:
     - ./verify/verify-flags-underscore.py
     - ./verify/verify-dashboard-format.sh
     - go get -d ./...
-    - travis_wait go build $(go list ./... | grep -v "/vendor/")
```

Comment:
Could you extract this change to a separate PR?

Reply:
Done, as #996